src/Umbraco.Web/PropertyEditors/GridPropertyEditor.cs

```diff
@@ -131,16 +145,41 @@ public override object ToEditor(Property property, IDataTypeService dataTypeServ
                 return grid;
             }
 
-            private GridValue DeserializeGridValue(string rawJson, out IEnumerable<GridValue.GridControl> richTextValues)
+            private GridValue DeserializeGridValue(string rawJson, out IEnumerable<GridValue.GridControl> richTextValues, out IEnumerable<GridValue.GridControl> mediaValues)
             {
                 var grid = JsonConvert.DeserializeObject<GridValue>(rawJson);
 
                 // Find all controls that use the RTE editor
-                var controls = grid.Sections.SelectMany(x => x.Rows.SelectMany(r => r.Areas).SelectMany(a => a.Controls));
+                var controls = grid.Sections.SelectMany(x => x.Rows.SelectMany(r => r.Areas).SelectMany(a => a.Controls)).ToArray();
```

Comment:
Only a nitpicky thing, i used to do ToArray as opposed to ToList by default, but it turns out that ToArray is slower and allocates more mem than ToList